src/content/docs/configuration/sharing.mdx

```diff
@@ -93,4 +93,8 @@ pull_request_rules:
 
   - Values in the `default` key will be merged and remote default values will
     apply to local configuration.
+
+  - If there is rule in any of the `pull_request_rules`, `queue_rules`, `priority_rules`, `partition_rules`
+    or `merge_protections` section with the same name in both the parent and child config, then the duplicated
+    rule in the child config will NOT be overwritten with rule from the parent config.
```

Comment:
child/parent wording is confusing here, I have no idea if the child is the one from `extends` or not.

Reply:
Modified it to describe which is which